lang/python38-devel/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           select 1.0
 
 name                python38-devel
 
-version             3.8.0b2
+version             3.8.0b3
 epoch               1
 revision            1
```

Comment:
The `revision` needs to be set back to 0 when increasing the `version`.

Reply:
Yup.